### PR TITLE
removing incorrect references

### PR DESCRIPTION
**Pull request by jmagak**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.2 

**Issue:** https://issues.redhat.com/browse/RHIDP-3990

**Link to docs preview:**

**Reviews:**
- [ ] SME: @ mention assignee
- [ ] QE: @ mention assignee
- [ ] Docs review: @ mention assignee
- [ ] Additional review: @mention assignee (by writer)

**Additional information:**

### CODE REVIEW

File: modules/getting-started/proc-add-source-control-rhdh-catalog.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [id='proc-add-source-control-rhdh-catalog_{context}']
 = Adding source control for Catalog in {product}
 
-To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a source control such as GitHub, GitLab, or BitBucket.
+To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a source control such as GitHub.
```
Comment (kim-tsao) [inline modules/getting-started/proc-add-source-control-rhdh-catalog.adoc:4]:
```suggestion
To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a supported source control such as GitHub.
```
TBH, this is the only source control but hopefully we can expand the list in the future

**Review by kim-tsao — APPROVED** [review]

**Review by its-mitesh-kumar — APPROVED** [review]

**Review by Gerry-Forde — APPROVED** [review]
/lgtm

**Comment by Gerry-Forde** [comment]
This PR has been reviewed by SME and Peer.

**Pull request merged**